libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4295,6 +4295,14 @@ void GCS_MAVLINK::handle_command_long(const mavlink_message_t &msg)
     mavlink_command_long_t packet;
     mavlink_msg_command_long_decode(&msg, &packet);
 
+#ifdef ENABLE_SCRIPTING
+    AP_Scripting *scripting = AP_Scripting::get_singleton();
+    if (scripting != nullptr && scripting->handle_command_long(packet, chan)) {
+        // Scripting is listening for this command don't process
```

Comment:
I wonder if we should still log in this case, but we wouldn't know the result of course...